osi_object.proto

```diff
@@ -264,6 +273,56 @@ message StationaryObject
             //
             COLOR_WHITE = 10;
         }
+
+        //
+        // \brief Attributes of type emitting structure
+        //
+        message EmittingStructureAttribute
+        {
+            // This message determines the range of the wavelength and its
```

Comment:
Add relationship to normal, i.e. angle is symmetrical across the normal, which is defined by the mounting position...